src/main/java/Bob.java

```diff
@@ -0,0 +1,221 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Bob {
+    public static void main(String[] args) {
+        String greet = " Hello! I'm Bob.\n"
+                + " What can I do for you?\n";
+
+        String exit = " Bye. Hope to see you again soon!";
```

Comment:
Consider changing this to a final String as the value of the greeting and exit phrase will not change